### PR TITLE
My portfolio - mobile version

### PR DESCRIPTION
**Pull request by NJ-2020**
Dear Reviewer,

This is a completion of the Mobile version skeleton.

I set up the portfolio and built and styled the mobile skeleton.

The mobile version consists of :
HTML and CSS grid.
Flexbox for the positioning.

I used Figma to apply the design.
I have also fixed all linter errors.

Please let me know your feedback for improvement.

Thanks!

### CODE REVIEW

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @NJ-2020 :handshake: :handshake: :handshake:, 

## Invalid Code Review Request

At this level, your project should have a headline including the navigation bar and your personal introduction, so what you have now on this Pull Request you have only works, the content from the previous project are missing, because that will make things awkward in the future milestones, then you need first of all to submit the headline milestone, and then this one will come after that!

_Action to take: Kindly work on [this project](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) first of all, when it gets approved then go ahead with this PR, So, for this time I highly recommend you to pause implementing the requested feedback in my previous review, first work on the aforementioned project_

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by descholar-ceo** [comment]
Hello @NJ-2020 , I saw that you might be having issues with `git push`, what issue do you have with it? Would you paste here the error message git is throwing ?

**Comment by NJ-2020** [comment]
Hi Mugirase,

I am confused actually.  I mistakenly cancelled the previous one and I redo
it again.
This is the link My portfolio - mobile version by NJ-2020 · Pull Request #1
· NJ-2020/Portfolio-Mobile-Version_w1d4 (github.com)
<https://github.com/NJ-2020/Portfolio-Mobile-Version_w1d4/pull/1>

But you seem to still review my old one.  I do not know how to sort it out.
Do you think we can have a chat on zoom?

Many thanks

Regards
Novell

**Comment by NJ-2020** [comment]
Hi Mugirase,

I am confused actually.  I mistakenly canceled the previous one and I redo
it again.

I don't really understand why you ask me to submit my Previous Project first before this project and that the first one is already approved?
Do you think we can have a chat on zoom?

Many thanks

Regards
Novell

**Comment by descholar-ceo** [comment]
> Hi Mugirase,
> 
> I am confused actually. I mistakenly canceled the previous one and I redo
> it again.
> 
> I don't really understand why you ask me to submit my Previous Project first before this project and that the first one is already approved?
> Do you think we can have a chat on zoom?
> 
> Many thanks
> 
> Regards
> Novell

Hello @NJ-2020 , yes please

File: index.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                <img src="dropright" alt="" />
+              </li>
+              <li class="skills">
+                <p><b>Skills</b></p>
+                <img src="dropright" alt="" />
+              </li>
+            </ul>
+          </section>
+        </li>
+      </ul>
```
Comment (smcommits) [inline index.html]: - [ ] **You are using a `ul` tag to group the `My-Project` section and `About Myself` section. Please separate them into their own section tags as they are two different sections on the page.**

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @NJ-2020, 

### You are doing a great job.
### There are some issues that you still need to work on. You are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


 ## Required Changes ♻️

- [ ] **The overall layout of the application is broken. Please fix it so that it matches the original**

Cloned Page|Original Design
:----------------:|:-------------------:
Screen Shot 2021-06-06 at 00 52 16-fullpage|image image

As you can see in the screenshot above, the content of your website is not properly aligned. It is overflowing out of the page. Also, the content is stacked vertically in the original design. Whereas in the cloned version, some elements are vertically occurring. Please fix this and make sure that the cloned page appears very close to the original design. 

_Please check the inline comments below._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NJ-2020** [comment]
Hi Anas,

I hope you are well.
I have done the changes as needed.
However, about your request regarding ul tag to group the My-project and About Myself, I don't understand what do you mean.
But what I noted, that they are already grouped in accordance with their sections.

I hope I have done as per your request.
Your feedback is welcome.

Thanks
Novell

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @NJ-2020 , 

Great work with making all the changes requested by previous TSEs. 
All content aligned well now. :+1: 

Your project is complete!  
Congratulations! 🎉


You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NJ-2020** [comment]
Thanks so much, Sir! 🥇

**Pull request merged**